src/test/java/pl/wilki/battleship/board/BoardTest.java

```diff
@@ -11,6 +11,69 @@
  */
 @Test
 public class BoardTest {
+
+  @DataProvider
+  public static Object[][] newDefaultBoardFieldsCheck() {
```

Comment:
I have no idea what does this DataProvider, you should name it like "fieldsTypeToCheckDataProvider"